src/index.ts

```diff
@@ -1767,6 +1777,13 @@ export interface Ketch {
    */
   registerStorageProvider(policy: StorageOriginPolicy, provider: StorageProvider): Promise<void>
 
+  /**
+   * Sets subscriptions
+   *
+   * @param subscriptions SetSubscriptionsRequest
+   */
+  setSubscriptionsRequest(subscriptions: SetSubscriptionsRequest): Promise<void>
```

Comment:
`setSubscriptions`

Also need to add a `getSubscriptions` taking a `GetSubscriptionsRequest` and returning `GetSubscriptionsResponse`.